Packs/AccentureCTI/Scripts/FormatACTIURL/FormatACTIURL.yml

```diff
@@ -1,7 +1,6 @@
 commonfields:
```

Comment:
`FormatACTIURL.py` is missing in `Packs/AccentureCTI/Scripts/FormatACTIURL`

Reply:
@kgal-pan I might be wrong, but I see `FormatACTIURL.py` under `Packs/AccentureCTI/Scripts/FormatACTIURL`